### ISSUE TITLE
French audio check doesn't cover some releases

### ISSUE THREAD

**Issue by andy3469** [Sonarr/Sonarr#7679]
### Is there an existing issue for this?

- [x] I have searched the existing open and closed issues

### Current Behavior

Currently "FRE" and "FRA" aren't recognized as French.

### Expected Behavior

Since the current regex only detects a fraction of the real life releases properly, i would expect it to be changed so that it does detect "FRE" and "FRA" as French.

### Steps To Reproduce

Show Name S01 Eng Fre Ger Ita Spa Cze Jpn 2160p BluRay Remux DV HDR HEVC Atmos SGF
Show Name.S01.Eng.Fre.Ger.Ita.Por.Spa.2160p.WEBMux.DV.HDR.HEVC.Atmos-SGF
Show.Name.S02E10.Episode.Title.French.DL.BD.x264-TVS
Show Name S01 Eng Fre Ger Ita Por Spa 2160p WEBMux HDR HEVC DDP SGF
Show Name S01 KOREAN ENG FRA GER ITA SPA MULTI 2160p NF WEB DL DDP5 1 DV HDR x265 Atmos MassModz
Show.Name.S02E09.EpisodeName.French.DL.BD.x264-TVS

### Environment

```markdown
- OS: Ubuntu 24.04
- Sonarr: 4.0.13.2932
- Docker Install: Yes
- Using Reverse Proxy: Yes
- Browser: 
- Database: Sqlite 3.48.0
```

### What branch are you running?

Main

### Trace Logs?

```
2025-02-25 22:29:21.6|Debug|Parser|Parsing string 'Severance.S01.Eng.Fre.Ger.Ita.Por.Spa.2160p.WEBMux.DV.HDR.HEVC.Atmos-SGF'
2025-02-25 22:29:21.6|Debug|Parser|Episode Parsed. Severance - Season 01 
2025-02-25 22:29:21.6|Debug|Parser|Languages parsed: English, Italian, Spanish
2025-02-25 22:29:21.6|Debug|QualityParser|Trying to parse quality for 'Severance.S01.Eng.Fre.Ger.Ita.Por.Spa.2160p.WEBMux.DV.HDR.HEVC.Atmos-SGF'
2025-02-25 22:29:21.6|Debug|Parser|Quality parsed: WEBRip-2160p v1
2025-02-25 22:29:21.6|Debug|Parser|Release Group parsed: SGF
2025-02-25 22:29:21.6|Debug|AggregateLanguages|Selected languages: English, Italian, Spanish

```

### Anything else?

This is the same kind of issue as [the one for German language](https://github.com/Sonarr/Sonarr/issues/7635) and it's already implemented on [Radarr](https://github.com/epmt7w3ugk/Radarr/blob/develop/src/NzbDrone.Core/Parser/LanguageParser.cs#L25) since 2021

**Comment by andy3469** [comment]
@markus101 I updated the issue with the Trace logs. Still I don't really know if it's a bug or a feature request.
